docs/threads/introduction.md

```diff
@@ -4,87 +4,89 @@ hero_img: /images/threads-hero.png
 
-Everything above just looks like a Database, so what's a Thread? ThreadsDB combines the storage and management of data (the Database) with networking, access control, and replication over IPFS using the Threads Protocol. The Threads protocol has been extensively documented in the [whitepaper](https://docsend.com/view/gu3ywqi), but in short, Threads use private-key encryption to manage both security and identity among multiple parties that can access or edit the same Database.
+By default, a ThreadDB will connect with a local `go-threads` Threads Daemon. See the [`go-threads`](https://github.com/textileio/go-threads) for details on getting started with local development. The Threads Daemon may be helpful to developers that aim to build their own Thread Services, host replication services, or test advanced Thread usage. The Threads Daemon (`threadsd`) is provided as an installable binary with every [release of Threads](https://github.com/textileio/go-threads/releases).
 
-Jump to [Thread Networking](#networking) to read more.
+Alternatively, it is possible to connect with a remote daemon by specifying the networking component of the Database ([See more](#replication-with-the-hub) for details on connecting to hosted remote services).
 
-## Replication with the Hub
 
-ThreadsDB has been designed to support trustless peers on the network to provide services that improve or enhance performance and experience for end-users. [The Hub](/hub/introduction) offers Thread Services for relay, replication, and backup that you can add for your users in a couple of minutes. 
+```typescript
+const db = new Database() // Uses a level datastore by default
+```
 
-### Connect to the Hub
+Next, we simply start the database, and we are ready to take action. Here, we are explicitly providing an 'existing' ThreadID. But default, a random ThreadID will be used. See the doc-strings for `Database.open` for further options.
 
-1. [Create an Account](/hub/accounts#account-setup)
-2. [Create an App Token](/hub/app-apis#api-access)
-3. <a href="https://textileio.github.io/js-textile" target="_blank">Add the Textile Library to your App</a>
 
-## ThreadsDB
+```typescript
+const threadID = ThreadID.fromRandom()
+await db.open({ threadID })
+console.log(db.threadID.toString())
+```
 
-### Basic Usage
+### Collections
 
-#### Databases
+To handle different data structures, a Database contains Collections, each of which are defined by a [json-schema.org schema](https://json-schema.org/). These schemas define the 'shape' of Collection Instances. Collections implement a Store with [JSON Patch](https://github.com/Starcounter-Jack/JSON-Patch) semantics by default, but will be made to support other types (CRDT-driven documents for instance) in the future (some of which are already under active development). Ultimately, a Collection is a single document store with a set of APIs to make it feel like a *local database table*.
```

Comment:
I think this one skips to explaining what makes up a collection without explaining what a collection is.  Maybe

`The database stores items in Collections`.

'Collections are similar to Tables in other databases' could help a lot.

Reply:
I think something like this is mentioned above in the intro? But repeating it here for emphasis.